### ISSUE TITLE
Remove the timestamp from the baseline files

### ISSUE THREAD

**Issue by BraisGabin** [arturbosch/detekt#1524]
Right know the baselines is something like this:
```xml
<SmellBaseline>
  <Blacklist timestamp="1549965557758"></Blacklist>
  <Whitelist timestamp="1550827039946">
    <!-- ... -->
  </Whitelist>
</SmellBaseline>
```

I think that the timestamp provides little to no value and generates a lot of git conflicts. Is this value necessary? Can we remove it or at least make it configurable?

**Comment by arturbosch** [comment]
sounds reasonable. I haven't used it at all :).

**Comment by cesards** [comment]
Has the fix been released? We are really interested in the fix since we get frequent merge conflicts due timestamps, but I haven't seen a changelog so I'm wondering since it keeps generating
```
<Blacklist timestamp="1554408898707"></Blacklist>
<Whitelist timestamp="1554408898707">
```
in the latest released version: `1.0.0-RC14`

Thank you in advance!

**Comment by arturbosch** [comment]
Not yet sorry! Please just wait a little longer for RC15 :)
